src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { create } from "react-test-renderer";
+import { shallow } from "enzyme";
+import { Button } from "./Button";
+
+it("renders correctly with all props", () => {
+  const component = create(
+    <Button
+      id="1234"
+      className="class1 class2"
+      isStartButton
+      name="Testname"
+      type="button"
+      disabled
+      onClick={(): void => {}}
+    >
+      Testchildren
+    </Button>
+  );
+
+  expect(component).toMatchSnapshot();
+});
+
+it.only("calls the onClick prop when clicked", () => {
```

Comment:
```suggestion
it("calls the onClick prop when clicked", () => {
```

Please take care with your tests.